src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,103 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
+
+    margin-bottom: unit(45px / @base-font-size-px, em);
+
+    .respond-to-min(1024px, {
+        margin-top: unit(60px / @base-font-size-px, em);
+    });
+
+    .image-container {
```

Comment:
Okay, refactored this in https://github.com/cfpb/cfgov-refresh/commit/0c2e3ed9967b5a4d8b07c98f9e1a92fdf20d94dc and added a `media-stack` BEM block for defining vertically stacked media objects.